packages/pricing-calculator/src/utils/slider.utils.ts

```diff
@@ -0,0 +1,60 @@
+type Interval = {
+  minimum: number;
+  maximum: number;
+  step: number;
+};
+
+type Options = {
+  controlPosition: number;
+  currentIndex: number;
+  stepDimension: number;
+  interval: Interval;
+};
+
+export const calculateIntervalValue = ({
```

Comment:
As we discussed - for future re-usage we could move it to `@keen.io/ui-core` package and export as slider utils.